pyproject.toml

```diff
@@ -1,3 +1,53 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aiobotocore"
+version = "2.13.3"
+description = "Async client for aws services using botocore and aiohttp"
+readme = "README.rst"
+requires-python = ">=3.8"
+authors = [
+    { name = "Nikolay Novik", email = "nickolainovik@gmail.com" },
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Natural Language :: English',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Environment :: Web Environment',
+    'Framework :: AsyncIO',
+]
+
+dependencies = [
+    # NOTE: When updating botocore, always keep `project.optional-dependencies` aligned
+    "botocore>=1.34.70,<1.34.163",
+    "aiohttp>=3.9.2,<4.0.0",
+    "wrapt>=1.10.10, <2.0.0",
+    "aioitertools>=0.5.1,<1.0.0",
+]
+
+[project.optional-dependencies]
+awscli = [
+    "awscli>=1.32.70,<1.33.45",
+]
+boto3 = [
+    "boto3>=1.34.70,<1.34.163",
+]
+
+[project.urls]
+Repository = "https://github.com/aio-libs/aiobotocore"
+PyPI = "https://pypi.org/project/aiobotocore/"
```

Comment:
These links are displayed on PyPI, in the sidebar. Perhaps, the project page shouldn't link to itself?